perfkitbenchmarker/pint_wrapper.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wrapper module for the pint library."""
+
+import pint
+
+# Pint requires one global UnitRegistry for the entire program, so we
+# create it here.
+ureg = pint.UnitRegistry()
```

Comment:
Moving it to `__init__.py` SGTM. If the usage gets any more complex (e.g., loading custom units), it can be broken out into a separate file and imported in `__init__.py` to preserve compatibility. Agree on the more descriptive variable name.